packages/compass-indexes/src/stores/create-index.js

```diff
@@ -47,7 +47,9 @@ const configureStore = (options = {}) => {
     });
 
     localAppRegistry.on('open-create-index-modal', () => {
-      track('Index Create Opened');
+      track('Index Create Opened', {
```

Comment:
Should we have this in the `create-index-modal` instead for consistency with how we do with search indexes then? Similar comment with the `Screen` event duplication as commented on above then.

Reply:
I aligned the tracking and will follow up with product if we can remove the duplicate events. If agreed, i'll clean it in a follow up :)